package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@tech-matters/hrm-base",
+  "version": "1.0.0",
+  "description": "",
+  "author": "",
+  "license": "ISC",
+  "workspaces": [
+    "packages/*",
+    "hrm-service",
+    "jobs/*"
+  ],
+  "dependencies": {
+    "@aws-cdk/aws-iam": "^1.176.0",
+    "@aws-cdk/aws-lambda": "^1.176.0",
+    "@aws-cdk/aws-lambda-event-sources": "^1.176.0",
+    "@aws-cdk/aws-lambda-nodejs": "^1.176.0",
+    "@aws-cdk/aws-s3": "^1.176.0",
+    "@aws-cdk/aws-sqs": "^1.176.0",
+    "@aws-cdk/aws-ssm": "^1.176.0",
+    "@aws-cdk/core": "^1.176.0"
+  },
+  "devDependencies": {
+    "@tsconfig/node16": "^1.0.3",
+    "@types/aws-lambda": "^8.10.72",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^14.14.26",
+    "@types/node-fetch": "^2.5.10",
+    "@types/redis": "^2.8.29",
+    "@typescript-eslint/eslint-plugin": "^5.40.1",
+    "@typescript-eslint/parser": "^5.40.1",
+    "aws-cdk": "^2.45.0",
+    "aws-cdk-local": "^2.15.0",
+    "aws-sdk": "^2.1231.0",
+    "eslint": "^8.2.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^16.1.4",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-import-resolver-typescript": "^2.7.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jest": "^23.8.2",
+    "eslint-plugin-prettier": "^3.4.1",
+    "jest": "^28.1.3",
+    "jest-config": "^28.1.3",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^1.19.1",
+    "ts-jest": "^28.0.8",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
+  },
+  "scripts": {
+    "build": "tsc -b",
+    "build-and-start": "run-s build start",
+    "build-and-start:service": "run-s build start:service",
+    "start": "run-s start:cdk start:service",
+    "start:cdk": "run-s localstack:init",
+    "start:service": "npm run -w hrm-service start",
+    "docker:build": "npm run docker:build --workspaces --if-present",
+    "docker:compose:down": "docker-compose -p hrm down",
+    "docker:compose:logs": "docker-compose -p hrm logs -f",
+    "docker:compose:up": "run-s docker:compose:*:up",
+    "docker:compose:db:up": "docker-compose -p hrm -f hrm-service/docker-database/docker-compose.yml up -d",
```

Comment:
Moving these to the top level and specifying `-p hrm` runs all composer assets in a single project and therefor a single docker network so that they communicate using docker service name resolution.